pybop/_optimisation.py

```diff
@@ -40,27 +40,24 @@ class Optimisation:
     def __init__(
         self,
         cost,
```

Comment:
I think we want users to overwrite the initial conditions, right? This modification should keep that open.

```suggestion
        cost,
        x0=None,
```